server/server_test.go

```diff
@@ -479,6 +480,45 @@ var _ = Describe("Server package test", func() {
 			testURL("DELETE", serverPluralURL+"/"+serverID, memberTokenID, nil, http.StatusUnauthorized)
 			testURL("DELETE", serverPluralURL+"/"+serverID, adminTokenID, nil, http.StatusNoContent)
 		})
+
+		Context("Visiblity of properties", func() {
+			const (
+				resourceType   = "visible_properties_test"
+				resourceID     = "resource_id"
+				visibleTokenID = "visible_token"
+				hiddenTokenID  = "hidden_token"
+			)
+
+			var url = visibilityTestPluralURL + "/" + resourceID
+
+			BeforeEach(func() {
+				testResource := map[string]interface{}{
+					"id": resourceID,
+					"a":  "a",
+					"b":  "b",
+				}
+				testURL("POST", visibilityTestPluralURL, adminTokenID, testResource, http.StatusCreated)
+			})
+
+			It("Should see visible fields", func() {
+				res := testURL("GET", url, visibleTokenID, nil, http.StatusOK)
```

Comment:
please add tests for create and update when `hiddenProperties` are non-empty in create and update policies

Reply:
I've added tests for create and update.

Reply:
@marcin-ptaszynski @zimnx @p-kozlowski I've applied all the comment please review again.